### DISCUSSION TITLE
Installation Question for Python 3.8.9

### DISCUSSION THREAD

**Discussion by fhhowdy** [pyproj4/pyproj#1003] [Q&A]
I'm setting up a new computer and ran into an issue when installing pyproj on Python 3.8.9.

The new laptop is a Mac (using one of their arm chips, but I'm not sure that it matters).

Any suggestions would be greatly appreciated! 

When I run:
`pip install pyproj`

I get the following result, which proceeds to loop indefinitely:

Collecting pyproj
  Downloading pyproj-3.3.0.tar.gz (217 kB)
     |████████████████████████████████| 217 kB 141 kB/s            
  Installing build dependencies ... done
  Getting requirements to build wheel ... error
  ERROR: Command errored out with exit status 1:
   command: /Users/user/PycharmProjects/epriGISMask/venv/bin/python /Users/user/PycharmProjects/epriGISMask/venv/lib/python3.8/site-packages/pip/_vendor/pep517/in_process/_in_process.py get_requires_for_build_wheel /var/folders/1j/1dcyb6f57fj4064zvnrypcn80000gn/T/tmp0e379_2p
       cwd: /private/var/folders/1j/1dcyb6f57fj4064zvnrypcn80000gn/T/pip-install-m2hkbbj8/pyproj_6950c8bb491d45459307024922ab1f10
  Complete output (1 lines):
  proj executable not found. Please set the PROJ_DIR variable. For more information see: https://pyproj4.github.io/pyproj/stable/installation.html
  ----------------------------------------
WARNING: Discarding https://files.pythonhosted.org/packages/e4/36/c151d658ca1a1ccfd5ed82ac3b41d13c36cbd0687da97ac1beaeb3381fa8/pyproj-3.3.0.tar.gz#sha256=ce8bfbc212729e9a643f5f5d77f7a93394e032eda1e2d8799ae902d08add747e (from https://pypi.org/simple/pyproj/) (requires-python:>=3.8). Command errored out with exit status 1: /Users/user/PycharmProjects/epriGISMask/venv/bin/python /Users/user/PycharmProjects/epriGISMask/venv/lib/python3.8/site-packages/pip/_vendor/pep517/in_process/_in_process.py get_requires_for_build_wheel /var/folders/1j/1dcyb6f57fj4064zvnrypcn80000gn/T/tmp0e379_2p Check the logs for full command output.

**Comment by snowman2** [comment]
What version of pip are you using to install pyproj?

**Reply by fhhowdy** [reply]
running `pip -V`:

pip 21.3.1 from /Users/user/PycharmProjects/epriGISMask/venv/lib/python3.8/site-packages/pip (python 3.8)

**Reply by snowman2** [reply]
Hmm, that version seems fine. I wonder if it has to do with the arm chip as you mentioned earlier. You could try using conda to install pyproj.

**Reply by fhhowdy** [reply]
I got a similar message when I tried the 3.9 installer from within conda.  However, backing the version down to 3.6 created an environment where everything worked.  I haven't created all of the different versions to see at which version the error starts appearing, but I'll try that in the next day or so.  For now, I'm up and running.
